docs/creating-pdf-documents.md

```diff
@@ -107,6 +129,14 @@ exports:
     output: exports/my-document.pdf
 ---
 ```
+Finally, build the PDF output with Typst using the following command:
+
+```bash
+myst build --typst
+```
+
+You can use [document frontmatter](./frontmatter.md) to control various aspects of your Typst outputs.
```

Comment:
Out of curiosity, is this recommended? `myst build --typst` will probably only work if you have a `myst.yml` file, so isn't that then the place to put the options?

Reply:
I am not sure - I think a lot of mystmd originated with the assumption that you were working in individual documents / articles, and thus put frontmatter into the `.md` file. The move into the `executablebooks/` org brought another major use-case: multi-page books, and has significantly boosted the importance of `myst.yml`. But I feel like there are still a lot of legacy mentions of article-focused workflows that don't hold anymore.

My intuition is that we should just merge this one for now since I don't really know the answer to your question. I am happy to try and brainstorm an answer to your question w/ you as a follow up.